src/screens/Errors/GitNotFound.tsx

```diff
@@ -22,19 +21,25 @@ export const GitNotFound = () => {
         <Stack>
           <div>
             <VisualTestsIcon />
-            <Heading>Visual tests</Heading>
+            <Heading>Setup a Git repository</Heading>
             <Text center muted>
-              Catch bugs in UI appearance automatically. Compare image snapshots to detect visual
-              changes.
+              Chromatic requires Git to associate test results with commits and branches.
```

Comment:
Not sure but you might want to link to this:
https://www.chromatic.com/docs/visual-tests-addon/#git-addon

Reply:
this would be at least a way to bring users to the Chromatic website

Reply:
That link doesn't provide any more information so I don't think it helps.